src/SignalRServiceExtension/TriggerBindings/SignalRTriggerBindingProvider.cs

```diff
@@ -52,9 +52,9 @@ public async Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext
             var resolvedAttribute = GetParameterResolvedAttribute(attribute, parameterInfo);
             ValidateSignalRTriggerAttributeBinding(resolvedAttribute);
 
-            var accessKeys = _managerStore.GetOrAddByConnectionStringKey(attribute.ConnectionStringSetting).AccessKeys;
+            var accessKeys = _managerStore.GetOrAddByConnectionStringKey(resolvedAttribute.ConnectionStringSetting).AccessKeys;
```

Comment:
I think no need to change this. We want to original value of `ConnectionStringSetting` instead of the resolved connection string. Actually the `ConnectionStringSetting` is not an `[AutoResolve]` attribute.